rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/executor/BakePoller.groovy

```diff
@@ -120,6 +125,11 @@ class BakePoller implements ApplicationListener<ContextRefreshedEvent> {
                             if (!cancellationSucceeded) {
                               bakeStore.removeFromIncompletes(roscoInstanceId, statusId)
                             }
+
+                            // This will have the most up-to-date timestamp.
```

Comment:
The incomplete bakes are scoped by rosco instance id, and we only take this branch if the managing instance is not _this_ instance.